packages/utils/src/type.ts

```diff
@@ -241,10 +242,10 @@ export const concat = (deep: boolean, object1?: any, object2?: any): any => {
  * @param deep If true, the merge becomes recursive (aka. deep copy).
  * @param val The value(s) to clone
  */
-export const clone = (deep: boolean, val: any) => {
+export const clone: any = (deep: boolean, val: any) => {
```

Comment:
Weil sich der Linter da sonst bei mir beschwert:

> Function implicitly has return type 'any' because it does not have a return type annotation and is referenced directly or indirectly in one of its return expressions.ts(7024)

Ich verstehe zwar nicht, warum ich explizit `any` angeben muss (da das doch eh der Default sein sollte?). Aber `any` ist hier wohl auf jeden Fall der korrekte Rückgabewert.
Wenn du in den Code schaust: Falls `val` weder ein Array, noch ein Object, noch ein String ist, wird einfach `val` zurückgegeben. Und `val` ist vom Typ `any`.

Reply:
ah okay sorry das ist der rückkgabewert, ich dachte das wäre dann der type der methode

Reply:
Doch ist der type der methode, das müsste `export const clone = (deep: boolean, val: any): any => {` heißen

Reply:
Ah, okay.